app/_locales/en/messages.json

```diff
@@ -5008,18 +5008,11 @@
     "message": "Sudden market changes can cause failures. If the problem continues, reach out to MetaMask customer support."
   },
   "smartTransactionPending": {
```

Comment:
When a translation was done into other languages (which is the case with `smartTransactionPending`) and you want to change its value, you have to delete 'smartTransactionPending' from all files and create a new translation key with a different name and new value, only in the `/en` folder.

Reply:
Hey @dan437 – it appears ([slack](https://consensys.slack.com/archives/CTQAGKY5V/p1727805714803069?thread_ts=1704742270.501479&cid=CTQAGKY5V)) we can edit existing english text without deleting the key and creating a new one.